features/file_browser.feature

```diff
@@ -0,0 +1,18 @@
+Feature: File browser 
+  As a CSCE 120 GitHub organization member
+  So that I can see the File Directory of selected assignment
+  I want to be able to see the file browser
+
+  Background: Assignments in the GitHub organization
+    Given the following assignments exist:
+        | assignment_name | repository_name   |
+        | assignment1     | assignment1       |
+        | assignment2     | assignment2       |
```

Comment:
Why does there need to be multiple assignments in the background information?

Reply:
Removed them.